.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug to help us improve.
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide a *minimal* script to reproduce the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Error Output
+      description: Please indicate the output from the above script and what you expect the output to be.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: freud Version
+      description: What version of freud are you using?
+      placeholder: v2.10.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Python Version
+      description: What version of python are you using?
+      placeholder: v3.8.2
+  - type: dropdown
+    attributes:
+      label: System Platform
+      description: What platform are you using?
+      options:
+        - MacOS
+        - Linux
+  - type: dropdown
+    attributes:
+      label: Installation method
+      description: How did you install freud?
+      options:
+        - Download from PyPI
+        - Download from conda-forge
+        - built from source
```

Comment:
Containers are another option that I assume is reasonable common given the containers that @joaander maintains. Is there a way to include an "other" option where users can provide more info on alternate methods of installation? That would save us from having to maintain an exhaustive list here, just a sufficiently comprehensive collection of common methods.

Reply:
I can add a container option, but I think those 4 options form a comprehensive set. Can you think of another way a user could install/use freud that doesn't fall under those categories?

I can add these options as placeholders in a text area if we need to allow more flexibility.

Reply:
I'm fine with not adding anything other than containers. The containers (specifically `glotzerlab-software`) should be there.